src/models/user.ts

```diff
@@ -0,0 +1,53 @@
+import {Prisma, PrismaClient, User} from "@prisma/client";
+
+const prisma = new PrismaClient()
+
+async function createUser(email:string, username: string, password: string, firstName: string, lastName: string): Promise<void> {
+    try {
+        await prisma.user.create({
+            data: {
+                username,
+                email,
+                password,
+                firstName,
+                lastName
+            }
+        })
+        console.log('User created successfully')
+    } catch(error) {
+        console.log('Error creating user', error)
```

Comment:
this will swallow error so client gets ok response while user won't be created 
it is better to rethrow error so that router sends correct status